Instruments/python/merging_trees.py

```diff
@@ -0,0 +1,41 @@
+import uproot
+import numpy as np
+from array import array
+import ROOT 
+
+usage = 'merging_trees.py'
+ 
+import argparse
+parser = argparse.ArgumentParser(description='Merging two TTrees.')
+parser.add_argument('--files', required=True, type=str, help="Input the files names separated by a comma")  
+parser.add_argument('--out', required=True, type=str, help="Output ROOT file")
+parser.add_argument('--tree', required=True, type=str, help="Tree name")    
+parser.add_argument('--tree-out', required=False, type=str, default="merged_tree", help=" out tree name") 
```

Comment:
by default, tree_out should be equal to input tree name. Set `default=None` and then in the code:
```python
if args.tree_out is None:
    args.tree_out = args.tree
```